docs/sources/write/shortcodes/index.md

```diff
@@ -333,6 +333,121 @@ compactor:
 
 {{< /collapse >}}
 
+## Guide
```

Comment:
Can we move this to be alphabetical in the page table of contents.

Reply:
Sure, if we don't move it to its own page I'll reorder.